lib/msal-node/test/client/PublicClientApplication.spec.ts

```diff
@@ -71,6 +75,27 @@ describe('PublicClientApplication', () => {
             });
     });
 
+    test('acquireTokenByRefreshToken', () => {
+        const msalConfig: ClientConfiguration = {
+            auth: {
+                clientId: TEST_CONSTANTS.CLIENT_ID,
+                authority: TEST_CONSTANTS.AUTHORITY,
+            }
+        };
+
+        const request: RefreshTokenRequest = {
+            scopes: TEST_CONSTANTS.DEFAULT_GRAPH_SCOPE,
+            refreshToken: TEST_CONSTANTS.REFRESH_TOKEN
+        };
+
+        const authApp = new PublicClientApplication(msalConfig);
+        authApp.acquireTokenByRefreshToken(request)
```

Comment:
You need to return this call, so that a promise is returned from the test. I'm guessing this test is ending before the assertions on line 95 are actually hit.

Reply:
You are right, the promises were not being executed. I updated the tests.